tests/unit-tests/test-class-lesson.php

```diff
@@ -1358,6 +1358,21 @@ public function testShouldShowLessonActions_WhenPrerequisiteFoundAndWasNotLesson
 		self::assertFalse( $result );
 	}
 
+	public function testShouldShowLessonActions_WhenHookSetsFalce_ReturnsFalse(): void {
```

Comment:
This test case name is somewhat misleading. In particular, the `WhenHookSetsFalse` part because in this test case we're testing both `true` and `false`. We could split these into 2 separate use cases to test each since there isn't much repetitive setup code.